onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -368,6 +368,7 @@ def collect_data(self, data_reader: CalibrationDataReader):
                 self.max_intermediate_outputs is not None
                 and len(self.intermediate_outputs) == self.max_intermediate_outputs
             ):
+                self.compute_data()
```

Comment:
>self.compute_data() [](http://example.com/codeflow?start=16&length=19)

could you please change condition "and len(self.intermediate_outputs) == self.max_intermediate_outputs" to "and len(self.intermediate_outputs) >= self.max_intermediate_outputs"